dist/windows/clementine.nsi.in

```diff
@@ -337,12 +374,9 @@ Section "Clementine" Clementine
   File "Qt5Network.dll"
   File "Qt5OpenGL.dll"
   File "Qt5Sql.dll"
-  File "Qt5Svg.dll"
   File "Qt5Widgets.dll"
   File "Qt5WinExtras.dll"
   File "Qt5Xml.dll"
-  File "Qt5XmlPatterns.dll"
```

Comment:
Pretty sure we use this somewhere.

Reply:
No. Nothing is using it. There are no DLL's linking to it at all, i've checked. The only thing that uses QtXml is liblastm, but even that library does not use QtXmlPatterns (a deprecated module). All other stuff in Clementine is using the new XML, Qt Core's QXmlStreamReader and QXmlStreamWriter classes.

Reply:
Ah, I was getting confused with all the myriad XML options in Qt.